base/pkg/entry.jl

```diff
@@ -553,6 +553,20 @@ function resolve(
         end
         rethrow(err)
     end
+    imported = String[]
+    for (pkg,(ver1,ver2)) in changed
+        if ver1 !== nothing && ver2 !== nothing
```

Comment:
Does `ver === nothing` mean that it was not updated?   I thought it meant just that the package was not using a tagged version.